docs/getting-started/ZE-system-reqs.md

```diff
@@ -16,18 +24,19 @@ Before installing Zowe Explorer, make sure that you meet the following requireme
 
 ### Integrated development environments: 
 
-- [VS Code](https://code.visualstudio.com/) 1.53.2+
-- [Eclipse Che](https://www.eclipse.org/che/)
-- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces)
-- [Theia](https://theia-ide.org/) 1.18+
+- [Red Hat CodeReady Workspaces](https://www.redhat.com/en/technologies/jboss-middleware/codeready-workspaces) 
+
+    :::note
+
+    Secure credentials are not supported in Red Hat CodeReady Workspaces as the keyrings are not unlocked by default.
```

Comment:
Not required, just a suggestion - since this line only refers to one environment, we can use "keyring" instead of "keyrings":

```suggestion
    Secure credentials are not supported in Red Hat CodeReady Workspaces as the keyring is not unlocked by default.
```

Reply:
Do we want to have a plug for the k8's vsc extension here for the code ready workspace? We could link to it's extender documentation if available or marketplace or blog until available?